### PR TITLE
chore: use skeleton templates

### PR DESCRIPTION
**Pull request by Fdawgs**
Use templates from https://github.com/fastify/skeleton to bring repo into line with other Fastify plugins.

#### Checklist

- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,12 @@
 # @fastify/session
 
-![Build Status](https://github.com/fastify/session/workflows/ci/badge.svg)
+![CI](https://github.com/fastify/session/workflows/CI/badge.svg)
 [![Coverage Status](https://coveralls.io/repos/github/fastify/session/badge.svg?branch=master)](https://coveralls.io/github/fastify/session?branch=master)
 [![NPM version](https://img.shields.io/npm/v/@fastify/session.svg?style=flat)](https://www.npmjs.com/package/@fastify/session)
-[![JavaScript Style Guide](https://img.shields.io/badge/code_style-standard-brightgreen.svg)](https://standardjs.com)
+[![Known Vulnerabilities](https://snyk.io/test/github/fastify/create-fastify/badge.svg)](https://snyk.io/test/github/fastify/create-fastify)
```
Comment (zekth) [inline README.md:6]:
```suggestion
[![Known Vulnerabilities](https://snyk.io/test/github/fastify/session/badge.svg)](https://snyk.io/test/github/fastify/session)
```

**Review by zekth — CHANGES_REQUESTED** [review]
Good initiative. One remark

File: README.md (hunk 1 above)
Comment (Fdawgs) [reply README.md]: Sorted, thanks @zekth

**Review by zekth — APPROVED** [review]
LGTM. test failing but it's not related.

**Pull request merged**